django/bosstiles/views.py

```diff
@@ -142,10 +144,15 @@ def get(self, request, collection, experiment, channel, orientation, resolution,
 
 
 
-        # Get interface to SPDB cache
-        cache = spdb.spatialdb.SpatialDB(settings.KVIO_SETTINGS,
-                                         settings.STATEIO_CONFIG,
-                                         settings.OBJECTIO_CONFIG)
+        # Get interface to SPDB or CVDB Mock cache
+        if resource.get_channel().is_cloudvolume():
```

Comment:
might be smart to refactor this at some point (doesn't have to be this PR) into something like

```python
cache = get_spatial_db(resource)
```

so that it becomes super duper easy to extend all of these in one place in the future, if we added, for example, zarrdb or something